package.json

```diff
@@ -119,6 +119,7 @@
     "focus-trap-react": "^8.11.3",
```

Comment:
I suggest you also install `"@types/mousetrap": "^1.6.15",` in the dev dependencies, it will be easier to work with TypeScript this way.

```
npm install --save-dev @types/mousetrap
```